huxley/api/views/school.py

```diff
@@ -66,7 +64,7 @@ def post(self, request, *args, **kwargs):
 class SchoolInvoice(generics.CreateAPIView):
```

Comment:
Is this class suppose to be here? It's not on the most updated version of master